packages/liveblocks-react/package.json

```diff
@@ -17,10 +17,22 @@
         "module": "./dist/index.mjs",
         "default": "./dist/index.js"
       }
+    },
+    "./suspense": {
```

Comment:
I remember this failing on some version of typescript, or maybe the ecosystem was not fully ready. Do you know if it's still the case?

Reply:
I think [this is the issue](https://github.com/liveblocks/liveblocks/pull/1641/commits/90eff09db6ecc011e6c41dbbfc8fb78670c6f6c4) you may remember. It was indeed a problem, but I found a solution for it yesterday! See the commit. Full context is in the link from the commit message.